tests/test_main.py

```
@@ -12,0 +1,326 @@
+            {'date': '2023-01-01', 'status': 'missing'},
+            {'date': '2023-01-02', 'status': 'ready'},
+            {'date': '2023-01-03', 'status': 'missing'},
+            {'date': '2023-01-04', 'status': 'ready'},
+            {'date': '2023-01-05', 'status': 'missing'},
+            {'date': '2023-01-06', 'status': 'ready'},
+            {'date': '2023-01-07', 'status': 'ready'},
+        ]
+        self.assertEqual(statuses, expected_statuses)
+
+if __name__ == "__main__":
+    logging.disable(logging.NOTSET) # Ensure all logs are enabled for direct test runs
```

Ruff (0.11.9)

Undefined name `logging`

(F821)